### DISCUSSION TITLE
Inline 'Ask Copilot' window is read only in Visual Studio 2022

### DISCUSSION THREAD

**Discussion by pboulton** [community/community#132548] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Sorry, maybe this is a newbie question because I just started with Copilot today but I don't understand why the inline  'Ask Copilot' window shown in the following screenshot is read only and therefore impossible to use:

![copilotissue](https://github.com/user-attachments/assets/f0da2fa7-eb18-4963-8c2a-dda731abb2d9)

Am I missing something here or is it a bug?  The 'Ask Copilot' panel works as expected, it's just the inline one that is read only.

My VS2022 version is 17.10.4 and my Copilot version is 0.2.889.30432.

**Comment by pboulton** [comment, marked as answer]
OP here - seems this is a current live bug as reported via the VS 2022 feedback hub.  Several users are reporting it.

**Reply by ayim** [reply]
Thanks! If you have any other feedback, please feel free to reach out directly to us

--
Aaron Yim
Product Manage for GitHub Copilot in Visual Studio
aayim@microsoft.com
